runtime/BUILD.gn

```diff
@@ -75,7 +75,10 @@ source_set("runtime") {
     sources += [ "ptrace_check.cc" ]
   }
 
-  public_deps = [ "//third_party/rapidjson" ]
+  public_deps = [
+    "//flutter/lib/ui",
```

Comment:
This includes a lib/ui header publicly, so it should always have been here anyways